plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/TrinoFileSystemCache.java

```diff
@@ -59,9 +60,12 @@
     private static final Logger log = Logger.get(TrinoFileSystemCache.class);
 
     public static final String CACHE_KEY = "fs.cache.credentials";
+    public static final String NO_WRAPPER_KEY = "trino.experiment.hive.no-wrapper-filesystems";
```

Comment:
@7c00 Can you please add a comment on why we need this config?
I think the intention is to use this config for any custom file system for which we want to avoid `FileSystemWrapper`.

Reply:
Added comments in TrinoFileSystemCache#createFileSystem